Common/Ubuntu/librealsense/stop_t265.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Stop t265 service
+
+if [ $(id -u) -ne 0 ]; then
+   echo >&2 "Must be run as root"
+   exit 1
+fi
+
+sudo systemctl stop t265
```

Comment:
nitpick, if you are root, you don't need sudo

Reply:
Thanks, fixed!